.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: python
 python:
   - "2.7"
+env:
+  - BITMESSAGE_HOME=/tmp
```

Comment:
Sad. I didn't planned to bound the tests to travis.

Reply:
I agree but I don't think it can be fixed without a lot of refactoring.